test/local/devices.js

```diff
@@ -314,10 +312,10 @@ describe('lib/devices:', () => {
             assert.equal(log.activityEvent.callCount, 1, 'log.activityEvent was called once');
             assert.equal(log.activityEvent.args[0][0].is_placeholder, true, 'is_placeholder was correct');
 
-            assert.equal(log.info.callCount, 1, 'log.info was called once');
-            assert.equal(log.info.args[0].length, 2);
-            assert.equal(log.info.args[0][0], 'device:createPlaceholder');
-            assert.deepEqual(log.info.args[0][1], {
+            assert.equal(log.info.callCount, 2);
```

Comment:
What is log.info.args[0] now that this is incremented, something related to sending a verification reminder?

Reply:
`log.info(redis.enabled)`